root/edit/details/merge_releases.tt

```diff
@@ -121,7 +121,7 @@
         [%~ END; END %]
     [%~ END ~%]
     </tbody></table>
-[%~ ELSIF edit.display_data.merges_are_calculated && !edit.is_open ~%]
+[%~ ELSIF !edit.is_open ~%]
     <p><strong>[% l('This edit does not store recording merge information and is closed, so no recording merge information can be shown.') %]</strong></p>
```

Comment:
Do we have a way of knowing whether no recordings needed merged in the first place? Otherwise all the stuff covered by ELSE below will become this as soon as the edit closes, no?

Reply:
That's a good point. I changed the conditions here to check if `recording_merges` is defined in the display data. Now, if it's not defined in the edit data (the edit is old), it's left undefined in the display data too so we can detect that.

But the answer is no, for old edits we can't easily know whether no recordings needed merging, since it doesn't store anything either way.

Reply:
But for edits newer than 0e41a26a69ab8f5d45b81f10ba321022791a13f0 yes, if the edit data has as an empty `recording_merges` array, then we know the recordings were already all merged.